docs/review_process.md

```diff
@@ -0,0 +1,144 @@
+- Writing comments (most of the time, it's a build status, either failure with logs or success).
+- Merging pull requests.
+- Closing issues (after merging pull requests with GitHub keywords).
+- Starting CI builds.
+- Assigning CI status (running/failed/successful).
+
+## Green build
+
+The first important prerequisite is ensuring your PR is green (build is successful).
+It requires a bit of patience, because there are many PRs are running, and we're building a lot of configurations for a numerous versions of libraries.
+Keep attention to the error messages from the bot, and address all the build failures.
+The bot tries to provide all the helpful information needed to understand and reproduce an issue, such as:
+
+- The profile that failed (in other words, the configuration: architecture, operation system, compiler, etc.)
+- Failed command line (it might have failed on early stages, like recipe syntax errors, hook errors or later stages, like build or test).
+- Logs containing the actual output of the build process (note that some logs like *configure.log* or *CMakeError.log* are not captured, only stdout/stderr).
+
+If you struggle to fix build errors yourself, you may want to ask for help from other users by mentioning (`@<user>`) individual users in the pull request comments.
+
+### Unexpected error
+
+Sometimes, build fails with `Unexpected error` message. This indicates an infrastructure problem, and usually it's unrelated to the changes within PR itself.
+Keep in mind conan-center-index is still in "early access program", and there are still some instabilities. Especially, as we're using lots of external services,
+which might be inaccessible (GitHub API, docker hub, etc.) and may result in intermittent failures.
+So, what to do once `Unexpected error` was encountered? You may consider re-running the build by closing your pull request, waiting 15 seconds, and the re-opening it again.
+
+Sometimes it's necessary to restart the build several times.
+If an `Unexpected error` persists, tag [@jgsogo](https://github.com/jgsogo) and [@danimtb](https://github.com/danimtb) asking for the help with CI.
+Alternatively, just [open a new issue](https://github.com/conan-io/conan-center-index/issues/new/choose).
+
+## Avoiding conflicts
+
+It's recommended to rebase your changes on top of the master branch to avoid conflicts.
+Right now, neither GitHub itself nor conan-center-bot notify about merge conflicts, so it's the contributor's responsibility to periodically check for the conflicts.
+Obviously, PRs that have merge conflicts are never merged, and all the conflicts have to be resolved first.
+
+## Draft
+
+Draft pull requests are also never merged. The same applies for PRs with the `WIP` keyword (stands for `Work in Progress`) in the title - GitHub considers them drafts as well.
+Once you're done with your changes, remember to convert from "Draft" to "Normal" pull request and remove WIP keyword from the title.
```

Comment:
Probably (not 100% sure) this is not accurate. When you open a PR with the `WIP` keyword it starts as a draft, but I think you can convert it into a regular PR without removing the WIP word. On our side, the bot doesn't take into account if the word `WIP` appears in the title or not.